sigs.yml

```diff
@@ -150,7 +150,7 @@
     - name: Severin Neumann
       github: svrnm
   - name: 'Semantic Conventions: LLM'
-    meeting: Every Wednesday alternating between 10:00 and 16:00 PT
+    meeting: Every Wednesday alternating between 10:00 and 23:00 PT
```

Comment:
We had this discussion now for a few other SIGs as well, would it make more sense to express the Asia/Europe meeting time in a different time zone that suits the people attending that meeting? PT means that daylight saving time is adjusted for people in PT, but people in areas without DST or a different DST will have their meeting times changed 2 (or 4) times.